internal/commands/prepare/prepare.go

```diff
@@ -181,6 +181,47 @@ func runPrepareVM(cmd *cobra.Command, args []string) error {
 		log.Printf("Timezone was set to %s!\n", tz)
 	}
 
+	dirsToMount := []string{}
+	if config.HostDir {
+		dirsToMount = append(dirsToMount, "hostdir")
+	}
+	if _, ok := os.LookupEnv(tart.EnvTartExecutorInternalBuildsDir); ok {
+		dirsToMount = append(dirsToMount, "buildsdir")
+	}
+	if _, ok := os.LookupEnv(tart.EnvTartExecutorInternalCacheDir); ok {
+		dirsToMount = append(dirsToMount, "cachedir")
+	}
+
+	for _, dirToMount := range dirsToMount {
+		log.Printf("Mounting %s...\n", dirToMount)
+
+		session, err := ssh.NewSession()
+		if err != nil {
+			return err
+		}
+		defer session.Close()
+
+		mkdirScript := fmt.Sprintf(
+			"mkdir -p /Users/%s/%s",
```

Comment:
What do you think about abstracting `Users/%s/%s` into separate variable to avoid mismatches between `mkdir` and `mount` invocations?

Reply:
SGTM. See [9e2887a](https://github.com/cirruslabs/gitlab-tart-executor/pull/65/commits/9e2887a03087482b9ae96e2bf8628ec59d28cd06)